index.html

```diff
@@ -86,22 +93,23 @@ <h1>I'm Naan. <br>Glad to see <br>you!</h1>
                         <div>2015</div>
                     </div>
                 </div>
-                <div class ="project_description">A daily selection of 
-                    privately personalized reads; no accounts or 
-                    sign-ups required.
+                <div class ="project_description">Experimental content creation feature 
+                    that allows users to add to an existing story over the course 
+                    of a day without spamming their friends.
                 </div>
                 <div class="tags">
                     <div>html</div>
+                    <div>ruby on rails</div>
                     <div>css</div>
                     <div>javascript</div>
```

Comment:
- [OPTIONAL] It is required to stick to the Figma design as much as possible. This includes font, font-weight, color, background-color, margin, padding among others. The padding on the language is not consistent as opposed to what we have on the figma design. Kindly look into it and fix it.

### Your Design
![your-stack](https://user-images.githubusercontent.com/26597432/221378028-fe8abc82-4457-4e20-bfb5-7ccfa2d85adb.png)

### Figma Design
![temp-stack](https://user-images.githubusercontent.com/26597432/221378038-00d09d50-d831-49df-b0d9-f117244ab9c9.png)